Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.12
```

Comment:
We should always support the latest py version.
And you are removing advertool for this consideration.